js/index.js

```diff
@@ -1 +1,54 @@
 // Your code goes here
+
+const homeContainer = document.querySelector(".container home");
+homeContainer.addEventListener("click", (event)=>{
+ console.log("home container was clicked!");
+});
+
+const navLink = document.querySelectorAll(".nav-link");
+
+navLink.addEventListener("click", ()=>{
+    event.target.style.color = "red";
+});
+
+const funBusImg = document.querySelector(".intro img");
+
+funBusImg.addEventListener("mouseleave", () => {
+    funBusImage.style.transform="rotate(5deg)";
+    console.log("hoppa");
+});
+introImg.addEventListener("dragstart", () => {
+    alert("This image cannot be dragged!");
+});
+
+const signMeButton = document.querySelectorAll(".btn");
+signMeButton.addEventListener("click", (event) => {
+    event.stopPropagation();
+    document.querySelector(".container home").style.backgroundColor = getRandomColor(); 
+	});
+
+    const AdventureImg = document.querySelector("adventure");
```

Comment:
You don't have a class or id with the name `adventure`. You're also missing the selector identifier (class or id). There also isn't an event listener.